guides/personal_mixed_skin.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: guides
+title: Creating your personal mixed skin
+description: "This guide teaches you how to create your own personal mixed skin."
+current: Guides | Mixing a skin
+---
+
+# Creating your personal mixed skin
+
+This guide will teach you how to create your own personal mix skin. You should know what SD and HD images are. If you don't, please read [this part](https://skinship.xyz/tutorial/introduction#hdsd-elements-aspect-ratios-and-resolution) of the skinning tutorial. You will also need to know the file names of assets in a skin. If you don't, please consult the [skinnable files list](https://docs.google.com/spreadsheets/d/1bhnV-CQRMy3Z0npQd9XSoTdkYxz0ew5e648S00qkJZ8/edit#gid=2074725196) for file names and other bits of information.
+
+Please note that more often than not, the different skins' assets will look wildly different from each other. This means that mixing is often best reserved for mixing gameplay elements, such as `cursor.png`, `cursortrail.png` etc.
+
+## Basic knowledge
+
+- HD and SD images - Your game will either render SD or HD images depending on your resolution. To check this, open settings in-game, using `CTRL + O`, navigate to the `GRAPHICS` section, and find the `Resolution` setting. If you have a vertical resolution (the second value) of 800 or above, your game is using HD images. If it is less, your game will be using SD images. As stated in the tutorial, HD images have a `@2x` suffix appended to them. Note that if you use an HD resolution and no HD image is present for an element, it will automatically fall back to the SD image, if it is present. As such, you should be mixing with both HD and SD images so that your mixed skin displays correctly for anyone who wants to use it.
+- .png and .jpg - osu! **only** supports .png images for skin elements, with **one** exception - `menu-background.jpg`, which can only be a .jpg image.

```

Comment:
would use "vs" rather than "and"

"must be" would be better than "can only be"